aten/src/ATen/native_parse.py

```diff
@@ -271,6 +271,13 @@ def has_sparse_dispatches(dispatches):
     return False
 
 
+def has_mkldnn_dispatches(dispatches):
```

Comment:
I don't think we need this, see https://github.com/pytorch/pytorch/pull/18144.

Reply:
Removed.